src/view.js

```diff
@@ -195,19 +199,51 @@ function startDragOrDraw(d) {
   };
 }
 
-function dragOrDraw(d, withDrag, withDraw) {
+function dragOrDraw(d, withDrag, withDraw, ctrl) {
   return function(e) {
     if ((e.shiftKey || util.isRightButton(e)) && d.drawable.enabled) withDraw(d, e);
-    else if (!d.viewOnly) withDrag(d, e);
+    else if (!d.viewOnly) withDrag(d, e, ctrl);
   };
 }
 
 function bindEvents(ctrl, el, context) {
   var d = ctrl.data;
-  var onstart = startDragOrDraw(d);
-  var onmove = dragOrDraw(d, drag.move, draw.move);
-  var onend = dragOrDraw(d, drag.end, draw.end);
-  var startEvents = ['touchstart', 'mousedown'];
+  var start = startDragOrDraw(d);
+  var move = dragOrDraw(d, drag.move, draw.move);
+  var end = dragOrDraw(d, drag.end, draw.end, ctrl);
+  var mousedown = false;
+
+  document.addEventListener('mousedown', function() {
+    mousedown = true;
+  });
+
+  document.addEventListener('mouseup', function() {
+    mousedown = false;
+  });
```

Comment:
that's adding event listeners everywhere chessground is loaded (every lichess.org page). Instead, add a chessground option to enable that functionality. So the normal chessground behavior is unchanged.